styles/variables-base.less

```diff
@@ -296,108 +305,107 @@
-@agate-slider-track-height: inherit;
+@agate-slider-track-height: 12px;
 @agate-slider-track-width: @agate-slider-track-height;
-@agate-translate-center: inherit;
-@agate-slider-bg-transform: inherit;
-@agate-slider-track-transform: inherit;
+@agate-translate-center: translate3d(-50%, -50%, 0);
+@agate-slider-bg-transform: none;
+@agate-slider-track-transform: none;
 
 // IncrementSlider
 // ---------------------------------------
-@agate-incrementslider-border-radius: inherit;
+@agate-incrementslider-border-radius: 0;
 @agate-incrementslider-horizontal-margin: @agate-slider-horizontal-margin;
-@agate-incrementslider-horizontal-slider-margin: inherit;
-@agate-incrementslider-horizontal-slider-padding: inherit;
-@agate-incrementslider-horizontal-button-margin: inherit;
-@agate-incrementslider-horizontal-button-decrement-border-radius: inherit;
-@agate-incrementslider-horizontal-button-decrement-border-width: inherit;
-@agate-incrementslider-horizontal-button-increment-border-radius: inherit;
-@agate-incrementslider-horizontal-button-increment-border-width: inherit;
+@agate-incrementslider-horizontal-slider-margin: 0 -1px;
+@agate-incrementslider-horizontal-slider-padding: ((@agate-slider-height - @agate-slider-track-height) / 2) (((@agate-slider-knob-width / 2) * @agate-slider-knob-resting-state-scale) + 3px);
+@agate-incrementslider-horizontal-button-margin: 0;
+@agate-incrementslider-horizontal-button-decrement-border-radius: @agate-slider-height 0 0 @agate-slider-height;
+@agate-incrementslider-horizontal-button-decrement-border-width: 0;
+@agate-incrementslider-horizontal-button-increment-border-radius: 0 @agate-slider-height @agate-slider-height 0;
+@agate-incrementslider-horizontal-button-increment-border-width: 0;
 @agate-incrementslider-vertical-margin: @agate-slider-vertical-margin;
-@agate-incrementslider-vertical-slider-margin: inherit;
-@agate-incrementslider-vertical-slider-padding: inherit;
-@agate-incrementslider-vertical-button-margin: inherit;
-@agate-incrementslider-vertical-button-decrement-border-radius: inherit;
-@agate-incrementslider-vertical-button-decrement-border-width: inherit;
-@agate-incrementslider-vertical-button-increment-border-radius: inherit;
-@agate-incrementslider-vertical-button-increment-border-width: inherit;
+@agate-incrementslider-vertical-slider-margin: 0;
+@agate-incrementslider-vertical-slider-padding: 0;
+@agate-incrementslider-vertical-button-margin: 0;
+@agate-incrementslider-vertical-button-decrement-border-radius: 0;
+@agate-incrementslider-vertical-button-decrement-border-width: 0;
+@agate-incrementslider-vertical-button-increment-border-radius: 0;
+@agate-incrementslider-vertical-button-increment-border-width: 0;
 
 // RadioItem
 // ---------------------------------------
 @agate-radioitem-separator-border-width: @agate-divider-border-width;
 @agate-radioitem-icon-border-width: @agate-button-border-width;
-@agate-radioitem-icon-border-radius: inherit;
+@agate-radioitem-icon-border-radius: 999px;
```

Comment:
How do you feel about setting a global skin corner radius variable and then using it throughout this file. That way, a skin could set the variable once and affect everything that is based on that border-radius uniformly. Ex: Electro rarely uses a fully round component, and could simply set this one variable once and appect all related components quickly.